pkg/chains/ton/proto_helpers.go

```diff
@@ -0,0 +1,118 @@
+package ton
+
+import (
+	"time"
+
+	types "github.com/smartcontractkit/chainlink-common/pkg/types/chains/ton"
+	valuespb "github.com/smartcontractkit/chainlink-common/pkg/values/pb"
+)
+
+func ConvertBlockIDExtToProto(block *types.BlockIDExt) *BlockIDExt {
+	if block == nil {
+		return nil
+	}
+	return &BlockIDExt{
+		Workchain: block.Workchain,
+		Shard:     block.Shard,
+		SeqNo:     block.SeqNo,
+	}
+}
+
+func ConvertBlockIDExtFromProto(pb *BlockIDExt) *types.BlockIDExt {
+	if pb == nil {
+		return nil
+	}
+	return &types.BlockIDExt{
+		Workchain: pb.Workchain,
+		Shard:     pb.Shard,
+		SeqNo:     pb.SeqNo,
+	}
+}
+
+func ConvertBlockToProto(block *types.Block) *Block {
+	if block == nil {
+		return nil
+	}
+	return &Block{
+		GlobalId: block.GlobalID,
+	}
+}
+
+func ConvertBlockFromProto(pb *Block) *types.Block {
+	if pb == nil {
+		return nil
+	}
+	return &types.Block{
+		GlobalID: pb.GlobalId,
+	}
+}
```

Comment:
nit/ if we can find the right terminology, these could be expressed more succinctly as constructors and methods:
```suggestion
func NewBlock(block *types.Block) *Block {
	if block == nil {
		return nil
	}
	return &Block{
		GlobalId: block.GlobalID,
	}
}

func (pb *Block) AsBlock() *types.Block {
	if pb == nil {
		return nil
	}
	return &types.Block{
		GlobalID: pb.GlobalId,
	}
}
```
It is just an awkward situation with the modified type (proto) being the one local to this package :thinking: